source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -568,7 +572,7 @@ Events
        * Current valid values are:
        *   - "stale":           The pool was cleared, making the connection no longer valid
        *   - "idle":            The connection became stale by being available for too long
-       *   - "error":           The connection experienced an error, making it no longer valid
+       *   - "connectionError": The connection experienced an error, making it no longer valid
```

Comment:
I renamed this constant as well to keep it consistent with `ConnectionCheckOutFailedEvent`.

Reply:
I'd prefer to keep it as `"error"` b/c:

1. that would require more changes to existing implementations, and might wind up breaking some current impls
2. Since it is part of a `ConnectionClosed` event, it is a little redundant. All errors at the `Connection` level are a `connectionError`.

Reply:
Makes sense. Done.